lib/puppet/provider/service/windows.rb

```diff
@@ -14,29 +14,30 @@
   defaultfor :operatingsystem => :windows
   confine    :operatingsystem => :windows
 
-  has_feature :refreshable
-  has_feature :configurable_timeout
+  has_feature :refreshable, :configurable_timeout, :configurable_logon_credentials
```

Comment:
I'd recommend `manages_xxx` for properties (not parameters) that can be managed, like we do for the user type: https://github.com/puppetlabs/puppet/blob/451f909c2e7965703805682a3f358917008e0625/lib/puppet/type/user.rb#L30-L61